src/utils/Firebase.tsx

```diff
@@ -42,10 +42,34 @@ export async function getUserDataForSpecificTask(jobID: string) {
     console.log(userDoc.data());
 }
 
+export async function createNewJob(userID: string, title: string, description: string, jobSkills: string[]) {
+    try {
+        const newJob = {
```

Comment:
You could run validation here, front end is doing validation but ideally you need to have safety in "backend" code. if this was backend code then you would validate and return the appropriate error message.